### DISCUSSION TITLE
Docker image based out of Debian 12 (bookworm)

### DISCUSSION THREAD

**Discussion by sekharvajjula** [tiangolo/uwsgi-nginx-docker#221] [Questions] [answered]
Do you have any near future plans to upgrade the image out of Debian 12 (bookworm).

**Comment by tiangolo** [comment, marked as answer]
The images are built automatically from the official base Python docker images, so when those are updated, this gets updated as well. :nerd_face:

**Comment by ptc-rdar** [comment]
@tiangolo I think there is an official bookworm images already in official python docker images, but the Docker files are pointing specifically to older release 
![image](https://github.com/user-attachments/assets/a2f971f1-18dd-41b0-9043-be96dff0c3a9)

**Reply by ptc-rdar** [reply]
If you change the base image to use
FROM python:3.12-bookworm

It should upgrade it
